application/src/main/java/org/thingsboard/server/controller/TelemetryController.java

```diff
@@ -487,13 +487,15 @@ public DeferredResult<ResponseEntity> deleteEntityTimeseries(
             @ApiParam(value = "A long value representing the end timestamp of removal time range in milliseconds.")
             @RequestParam(name = "endTs", required = false) Long endTs,
             @ApiParam(value = "If the parameter is set to true, the latest telemetry will be rewritten in case that current latest value was removed, otherwise, in case that parameter is set to false the new latest value will not set.")
-            @RequestParam(name = "rewriteLatestIfDeleted", defaultValue = "false") boolean rewriteLatestIfDeleted) throws ThingsboardException {
+            @RequestParam(name = "rewriteLatestIfDeleted", defaultValue = "false") boolean rewriteLatestIfDeleted,
+            @ApiParam(value = "If the parameter is set to true, the latest telemetry can be removed, otherwise, in case that parameter is set to false the latest value will not removed.")
```

Comment:
Could you improve the parameter description for clarity? Noone knows what is `latest value` in your case. Add some context. For example: 'The platform stores the latest arrived telemetry value (latest value) separately from the historical values. ... '

Reply:
Please improve the description of "rewriteLatestIfDeleted" parameter as well. I believe The order of parameters is wrong. It is hard to read about 'rewriteLatestIfDeleted' if you don't understand about 'deleteLatest' first